Documents/UseCasesForWebsite.txt

```diff
@@ -1,4 +1,26 @@
--  user wants to see the info page 
--  user wants to apply as artist 
-- user wants to donate 
--  user wants to have a prostictic worked on
\ No newline at end of file
+Actors: Visitor, Artist, Admin
+
+Visitor - View info page, View Gallery, Donate, RequestProstetic, Liking a Picture.
+A visitor would use this website for geting information or donate as amputee a prostetic or donate as a company.Also the visitors can visit the gallery and like one of the displayed art there.
+A visitor 
+- can also request that his/her prostetic is send to artist to make some art of it.
+- to donate a protestic he/she fill in a form and the admin of the site would contact the visitor. 
+- who is representing a company wants to donate a prostetic , also a form is filled in. 
+- goto the gallery page and like a prostic which is turn into art by a artist, he/she can like it so the prostetic is displayed earlier on the gallery page. 
+
+Artist - Apply to be part of this project so the artist can paint a prostetic.
+An artist could use this website for the same reasons as a regular user, but also the artist can fill in a form and send two pictures of art the artist make so the admin can look if the artist is good enough to 
```

Comment:
"An artist could use this website for the same reasons as a regular user". The sentence should end with that.
Continue describing each new case...